services/app-web/src/pages/QuestionResponse/UploadResponse/UploadResponse.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react'
+import {
+    GridContainer,
+    Form as UswdsForm,
+    FormGroup,
+    ButtonGroup,
+} from '@trussworks/react-uswds'
+import styles from '../UploadAnswers.module.scss'
```

Comment:
Looks like the PR is missing this css file.

Reply:
FYI I'm adding the class to QuestionResponse.module.scss and just sharing scss for both forms from t
here.